boa/environment.py

```diff
@@ -47,10 +47,15 @@ def __init__(self, fork_try_prefetch_state=False, fast_mode_enabled=False):
 
         self._gas_tracker = 0
 
+        self.nickname = "pyevm"
+
         self.evm = PyEVM(self, fast_mode_enabled, fork_try_prefetch_state)
 
     def set_random_seed(self, seed=None):
         self._random = random.Random(seed)
+    
+    def set_nickname(self, nickname):
```

Comment:
when would set_nickname be called?

Reply:
Maybe you have 2 ZKSync branches and you want to call one `zksync-testnet` and one `zksync-mainnet`

Reply:
yea but it is already set at construction, so i'm not sure why set_nickname would be called after construction

Reply:
If you’re in a Python shell and you setup zksync and called it zksync, but then realized you have 2 zksync chains, you’d have to just make two new ones with new names? Feels awkward to then have 3.

- zksync 
- Zksync testnet 
- Zksync mainnet